mobile/src/navigation/root/root.navigation.tsx

```diff
@@ -26,6 +26,8 @@ const Root: FC = () => {
   const hasUser = Boolean(user);
   const hasToken = Boolean(storage.get(StorageKey.ACCESS_TOKEN));
 
+  const initialScreen = hasUser ? RootScreenName.APP : RootScreenName.AUTH;
```

Comment:
```suggestion
```

Reply:
without this check we will always get to auth screen after app reload, even if user was authorized

Reply:
we should add something like this:
```
  const hasUser = Boolean(user)

  useEffect(() => {
    if(hasUser) {
      navigation.navigate(hasUser ? RootScreenName.APP : RootScreenName.AUTH);
    }
  }, [hasUser]);
```

Reply:
I've tried it earlier, but after app reload this provides such an warning/error
![Screenshot 2022-09-09 183922](https://user-images.githubusercontent.com/89215185/189388795-44a6cba7-cf94-492a-bc4b-4bcb630a31cb.png)

Reply:
Add additional flag isNavigationReady

Reply:
This should be in app component

Reply:
and this too
  ```typescript
  useEffect(() => {
    if (hasToken) {
      dispatch(authActions.loadCurrentUser());
    }
  }, [dispatch, hasToken]);
  ```

Reply:
Do you mean in src/components/app?

Reply:
Why the option with initialRouteName is not good enough?

Reply:
The main reason was that we need to await till user will be loaded, didn't notice that we have loader and there won't be such cases

Reply:
Just remove comment and that's it